suite-native/module-accounts/src/screens/AccountDetailScreen.tsx

```diff
@@ -65,10 +65,10 @@ export const AccountDetailScreen = memo(() => {
         if (account) {
             analytics.report({
                 type: EventType.AssetDetail,
-                payload: { assetSymbol: account.symbol, tokenSymbol },
+                payload: { assetSymbol: account.symbol, tokenSymbol: token?.symbol },
```

Comment:
I think we should change this also, so we are reporting a unique identifier to analytics as well. We need to put it in the analytics changelog in Notion though.

Reply:
Yeah, this is analytics specific and not technical improvement so we should handle this with analytics team separately.

Reply:
Ok, then please create an issue for that.